README.md

```diff
@@ -18,8 +18,18 @@ Add spree_ams to your Spree store's Gemfile:
 gem 'spree_ams', :github => 'hhff/spree_ams', :branch => '2-3-alpha'
 ```
 
+Make sure `config.action_controller.asset_host` is set in your environment config if not running in production.
```

Comment:
True.  Hmm.  I think we need to test if it's an absolute URL, then if not, test if asset host is set for the environment, then if not just pass the relative URL.